api/Navigator.json

```diff
@@ -281,21 +281,35 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
           "support": {
+            "webview_android": {
+              "version_added": "40"
+            },
             "chrome": {
               "version_added": "40"
             },
             "chrome_android": {
               "version_added": "40"
             },
-            "edge": {
-              "version_added": "17"
-            },
+            "edge": [
+              {
+                "version_added": "16",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Enable service workers"
+                  }
+                ]
+              },
+              {
+                "version_added": "17"
+              }
+            ],
             "edge_mobile": {
-              "version_added": "17"
+              "version_added": null
```

Comment:
Edge mobile tends to be the same as Edge desktop.

/cc @erikadoyle

Reply:
Then should we update it everywhere?

Reply:
I’m currently waiting for **Erika Doyle** for confirmation, but I expect version `17` to be correct.

Reply:
The flag was removed and SW works by default in EdgeHTML 17 builds, however *null* is fine for current mobile status. Although Fetch/SW is plumbed into the codebase, mobile is not explicitly supported/tested at this time.